pages/blog/_slug.vue

```diff
@@ -17,14 +17,30 @@
 
 <script>
 export default {
-  async asyncData ({ $content, params, error }) {
-    const [post] = await $content('blog', { deep: true })
-      .where({ dir: `/blog/${params.slug}` })
+  async asyncData ({ $content, params, error, app }) {
+    const currentLocale = app.i18n.locale
+    const isEnglish = currentLocale === 'en'
+    let [post] = await $content('blog', { deep: true })
+      .where({
+        dir: isEnglish ? `/blog/${params.slug}` : `/blog/${params.slug}/locales`,
+        slug: isEnglish ? 'index' : currentLocale
```

Comment:
Put the english file as any other lang file to keep it consistent.
```
-BlogName
    -image.jpg
    -locales
        -en.md <=
        -es.md
        -fr.md
```

So you endup with a code like this:
```suggestion
        dir: `/blog/${params.slug}/locales`,
        slug: currentLocale
```

Reply:
Done.